src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotInverso = {};
+
+  for (const value in robot) {
+    const robotValue = robot[value];
```

Comment:
```suggestion
  for (const property in robot) {
    const robotValue = robot[property];
```
Actually, [for...in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) iterates properties(keys) of object, so previous comment was only about changing robotKey to robotValue